web/src/components/storage/VolumeForm.jsx

```diff
@@ -310,6 +310,10 @@ const SizeAuto = ({ volume }) => {
       // TRANSLATORS: conjunction for merging two list items
                             volume.outline.sizeRelevantVolumes.join(_(", "))));
 
+  if (volume.outline.adjustByRam)
+    // TRANSLATORS: item which affects the final computed partition size
+    conditions.push(_("the amount of RAM in the system"));
```

Comment:
I wonder if worth mentioning _the strategy_ too. I mean, making it clear that will be the same size as RAM, the double, half...

Reply:
Hmmm, it probably would be more useful if we display the size of the RAM as part of the message. Users may not always have that in mind.

Reply:
I added a note about that at the PR description (and the corresponding Trello card for the future).